caproto/curio/server.py

```diff
@@ -200,6 +200,7 @@ def get_db_entry():
                         self.client_hostname, self.client_username,
                         command.data, command.data_type, command.metadata)
                 except Exception as ex:
+                    logger.exception('in hadle write')
```

Comment:
This is the first I've seen `logger.exception`. Super useful! Is this supposed to say "invalid write"?